packages/auth/src/core/index.ts

```diff
@@ -60,6 +60,35 @@ export function setPersistence(
 ): Promise<void> {
   return getModularInstance(auth).setPersistence(persistence);
 }
+
+/**
+ * Loads the reCAPTCHA configuration into the `Auth` instance.
+ *
+ * @remarks
+ * This will load the reCAPTCHA config, which indicates whether the reCAPTCHA
+ * verification flow should be triggered for a specific auth provider, into the
+ * current Auth session and affect future auth requests.
```

Comment:
```suggestion
 * This will load the reCAPTCHA config, which indicates whether the reCAPTCHA
 * verification flow should be triggered for each auth provider, into the
 * current Auth session.
```

Reply:
Thanks! Suggestions applied.